### PR TITLE
GRIP-22: Add support for booting LiveCD

### PR DESCRIPTION
**Pull request by zyoung51**
@RackHD/corecommitters

### CODE REVIEW

**Comment by yyscamper** [comment]
:+1:

File: data/profiles/boot-livecd.ipxe [hunk 1]
```diff
@@ -0,0 +1,4 @@
+kernel <%=repo%>/vmlinuz0 
+initrd <%=repo%>/initrd.img
+imgargs vmlinuz0 rootflags=loop initrd=initrd.img root=live:/<%=version%>.iso rootfstype=iso9660 !text !lang !ksdevice
```
Comment (yyscamper) [inline data/profiles/boot-livecd.ipxe]: How should the user put the ISO image? I didn't see the "root=live:/<%=version%>.iso" reference the <%=repo%>

Comment (zyoung51) [inline data/profiles/boot-livecd.ipxe]: setup_iso.py automates the setup of the ISO on disk.  The installation looks like: /LiveCD/isoname, Mirroring the other OS flows, 'version' is 'isoname' and repo is '/LiveCD/isoname'.

**Comment by jlongever** [comment]
+1

**Comment by anhou** [comment]
+1

**Pull request merged**